packages/clay-language-picker/src/index.tsx

```diff
@@ -0,0 +1,285 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Option, Picker} from '@clayui/core';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import {InternalDispatch, sub} from '@clayui/shared';
+import classNames from 'classnames';
+import React from 'react';
+
+type DisplayType = 'info' | 'secondary' | 'success' | 'warning';
+
+type Item = {
+	displayName?: string;
+	href?: string;
+	id: string;
+	label: string;
+	symbol: string;
+};
+
+type Labels = {
+	default: string;
+	option: string;
+	trigger: string;
+	translated: string;
+	translating: string;
+	untranslated: string;
+};
+
+type Translation = {total: number; translated: number};
+
+type Translations = {
+	[key: string]: Translation;
+};
+
+type Props = {
+	/**
+	 * Flag to indicate if the Picker is active or not (controlled).
+	 */
+	active?: boolean;
+
+	/**
+	 * Class names for the trigger.
+	 */
+
+	classNamesTrigger?: string;
+
+	/**
+	 * Flag to hide the text in the trigger.
+	 */
+
+	hideTriggerText?: boolean;
+
+	/**
+	 * The id of the component.
+	 */
+	id?: string;
+
+	/**
+	 * Labels for the component
+	 */
+	labels?: Labels;
+
+	/**
+	 * List of locales to allow localization for
+	 */
+	locales: Array<Item>;
+
+	/**
+	 * Callback for when the active state changes (controlled).
+	 */
+
+	onActiveChange?: InternalDispatch<boolean>;
+
+	/**
+	 * Callback that gets called when a selected locale gets changed
+	 */
+	onSelectedLocaleChange: (val: React.Key) => void;
+
+	/**
+	 * Exposes the currently selected locale
+	 */
+	selectedLocale: Item;
+
+	/**
+	 * Flag to show the picker in small size.
+	 */
+	small?: boolean;
+
+	/**
+	 * Path to the location of the spritemap resource.
+	 */
+	spritemap?: string;
+
+	/**
+	 * Translations provided to the component to be used and modified by it
+	 */
+	translations?: Translations;
+};
+
+const TranslationLabel = ({
+	defaultLanguage,
+	labels,
+	locale,
+	translation,
+}: {
+	defaultLanguage: Item;
+	labels: Labels;
+	locale: Item;
+	translation: Translation;
+}) => {
+	let displayType: DisplayType = 'warning';
+	let label = labels.untranslated;
+
+	if (locale.label === defaultLanguage?.label) {
+		displayType = 'info';
+		label = labels.default;
+	} else if (translation) {
+		const {total, translated} = translation;
+
+		if (total && total === translated) {
+			displayType = 'success';
+			label = labels.translated;
+		} else {
+			displayType = 'secondary';
+			label = sub(labels.translating, [translated, total]);
+		}
+	}
+
+	return (
+		<>
+			<span className="sr-only">
+				{sub(labels.option, [
+					locale.displayName || locale.label,
+					label,
+				])}
+			</span>
+
+			<ClayLabel
+				aria-hidden="true"
+				className="c-ml-3"
+				displayType={displayType}
+			>
+				{label}
+			</ClayLabel>
```

Comment:
Instead of using a div for the SR I think we could just add an `aria-label` on the ClayLabel with the text instead to avoid adding more elements to the DOM.

Reply:
Yep, agree with this. I have updated the `aria-label` of the `Option` component